static/app/components/buttonBar.tsx

```diff
@@ -3,14 +3,13 @@ import {css} from '@emotion/react';
 import styled from '@emotion/styled';
 import classNames from 'classnames';
 
-import type {ButtonProps} from 'sentry/components/core/button';
 import {StyledButton} from 'sentry/components/core/button';
 import type {ValidSize} from 'sentry/styles/space';
 import {space} from 'sentry/styles/space';
 
 type ButtonBarProps = {
   children: React.ReactNode;
-  active?: ButtonProps['barId'];
+  active?: string;
```

Comment:
L36-37 in this file
```ts
// We do not want to pass `barId` to <Button>`
const {barId, ...props} = childProps;
```

Reply:
We actually do that in different places, and then remove it... I will drop this in a separate PR